planetiler-benchmarks/src/main/java/com/onthegomap/planetiler/benchmarks/BenchmarkMbtilesWriter.java

```diff
@@ -68,10 +68,7 @@ public static void main(String[] args) throws IOException {
       Path outputPath = getTempOutputPath();
       try (var mbtiles = Mbtiles.newWriteToFileDatabase(outputPath, config.compactDb())) {
 
-        mbtiles.createTables();
-        if (!config.deferIndexCreation()) {
-          mbtiles.addTileIndex();
-        }
+        mbtiles.createTables(config.skipIndexCreation());
```

Comment:
One other thing I noticed in the mbtiles write benchmark is that it iterates through tile coordinates like this:

```java
    for (int z = 0; z <= 14; z++) {
      int maxCoord = 1 << z;
      for (int x = 0; x < maxCoord; x++) {
        for (int y = 0; y < maxCoord; y++) {
```

but the tile index order actually inverts Y - could you change to be:

```java
for (int y = maxCoord - 1; y > 0; y--) {
```